setup.py

```diff
@@ -20,18 +20,18 @@
 	install_requires=[
 	'scipy==1.5.4',
 	'matplotlib==3.3.3',
-	'tensorflow~=2.3.3',
-	'tensorflow-addons',
-	'dill',
+	'tensorflow==2.3.4',
```

Comment:
Can we keep the tensorflow version same as before?

Reply:
Sure, its just that when I run pip freeze on the Alpha2 version container, I see 2.3.4 was picked. I have now changed it back to 2.3.3